zebra/zebra_rib.c

```diff
@@ -3727,6 +3755,12 @@ static void early_route_meta_queue_free(struct meta_queue *mq, struct list *l,
 	}
 }
 
+static void rib_meta_queue_gr_run_free(struct meta_queue *mq, struct list *l,
+				       struct zebra_vrf *zvrf)
```

Comment:
Do we intend to free anything here?

Reply:
it's overrated if you ask me.   I've fixed it though